src/transaction/locator_sr.c

```diff
@@ -13766,19 +13768,21 @@ locator_multi_insert_force (THREAD_ENTRY * thread_p, HFID * hfid, OID * class_oi
 	        }
 
 	      for (size_t j = 0; j < recdes_array.size (); j++)
-	        {
+		{
```

Comment:
I think we need stats for `locator_multi_insert_force`:
stats for multi-insert
avg num of objects inserted to a page
...

You may separate the task from this.